web/src/components/FilingsCalendar.test.tsx

```diff
@@ -111,4 +149,38 @@ describe("<FilingsCalendar />", () => {
       )
     ).toBeInTheDocument();
   });
+
+  it("does not displays filings calendar as list when isRoadmap prop is true and formation date is undefined", () => {
```

Comment:
two thoughts:
- if we're introducing new tests for what happens when `isRoadmap` prop is true, then we should have a matching test for when it's false that describes that behavior
- I think it might be better if we renamed this something other than `isRoadmap`. It doesn't communicate to me what will change in the component when I switch the prop between true and false.